target/linux/ath79/dts/ar9330_teltonika_rut230.dts

```diff
@@ -0,0 +1,200 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9330.dtsi"
+
+/ {
+	compatible = "teltonika,rut230", "qca,ar9330";
+	model = "Teltonika RUT230";
+
+	chosen {
+		bootargs = "console=ttyATH0,115200 root=31:05 rootfstype=squashfs";
+	};
+
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &signal3;
+		led-failsafe = &signal3;
+		led-upgrade = &signal3;
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 22 GPIO_ACTIVE_HIGH>;
+			debounce-interval = <60>;
+		};
+
+		input {
+			label = "input";
+			linux,code = <BTN_0>;
+			gpios = <&gpio 16 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+
+		sim {
+			label = "SIM holder";
+			linux,code = <KEY_RFKILL>;
+			gpios = <&gpio 20 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		signal1 {
+			label = "rut230:green:signal1";
+			gpios = <&gpio 23 GPIO_ACTIVE_LOW>;
+		};
+
+		signal2 {
+			label = "rut230:green:signal2";
+			gpios = <&gpio 7 GPIO_ACTIVE_LOW>;
+		};
+
+		signal3: signal3 {
```

Comment:
Please include "led_" prefix for the DT label (as this is used in a global context):

`led_signal3: signal3 {`